.goreleaser.yml

```diff
@@ -12,6 +12,12 @@ release:
     - glob: ./manifest/charts/*
     - glob: ./manifest/yamls/*
 
+variables:
+  fetcherDescription: Fetcher is a very lightweight component and all of related logic is in fetcher package itself. Fetcher helps in fetching and uploading code and in specializing environments.
```

Comment:
Fetcher is a lightweight component used by environment and builder pods. Fetcher helps in fetch and upload of source/deployment packages and specializing  environments.

Reply:
Done